editor/src/components/editor/canvas-toolbar.tsx

```diff
@@ -425,6 +425,10 @@ export const CanvasToolbar = React.memo(() => {
     ? 'Not connected to room'
     : 'Comment Mode'
 
+  const commentButtonTestId =
+    roomStatus !== 'connected'
```

Comment:
Minor: Can invert this ternary to make it slightly simpler.

Reply:
inverted the ternary in ef31ae763be9b60bdeda032cef09314bc4555d3f